packages/twenty-front/src/modules/workflow/components/WorkflowEditActionFormServerlessFunction.tsx

```diff
@@ -31,7 +56,8 @@ export const WorkflowEditActionFormServerlessFunction = (
     props.action.settings.input.serverlessFunctionInput;
 
   const [functionInput, setFunctionInput] =
-    useState<Record<string, any>>(defaultFunctionInput);
```

Comment:
since you now have `inputSchema`, I would rename this one functionInputValue to be more explicit. What do you think?

Reply:
not sure about this since we use functionInput in back